lxd/auth/driver_common.go

```diff
@@ -37,6 +37,8 @@ type requestDetails struct {
 	isAllProjectsRequest bool
 	projectName          string
 	isPKI                bool
+	idpGroups            []string
```

Comment:
is this commit related or dependent on openfga driver?

Reply:
Yes, these aren't used by the TLS driver but are required for the OpenFGA driver.

Reply:
OK thanks